query/src/main/java/io/keen/client/java/RequestUrlBuilder.java

```diff
@@ -15,43 +16,45 @@
 class RequestUrlBuilder {
     // The API version string
     private final String apiVersion;
-    
+
     // The base URL, including the scheme and domain
     private final String baseUrl;
-    
+
     RequestUrlBuilder(String apiVersion, String baseUrl) {
         if (null == apiVersion || apiVersion.trim().isEmpty()) {
             throw new IllegalArgumentException("'apiVersion' is a required argument.");
         }
-        
+
         if (null == baseUrl || baseUrl.trim().isEmpty()) {
             throw new IllegalArgumentException("'baseUrl' is a required argument.");
         }
-        
+
         this.apiVersion = apiVersion;
         this.baseUrl = baseUrl;
     }
-    
+
     /**
-     * Get a format URL for an analysis request.
+     * Get a formatted URL for an analysis request.
      * 
      * @param projectId The project id
-     * @param analysisName The analysis name
+     * @param analysisPath The analysis url sub-path
+     *
      * @return The complete URL.
      * @throws KeenQueryClientException
      */
-    URL getAnalysisUrl(String projectId, String analysisName) throws KeenQueryClientException {
+    URL getAnalysisUrl(String projectId, String analysisPath) throws KeenQueryClientException {
         try {
-            return new URL(String.format(
-                    Locale.US,
-                    "%s/%s/projects/%s/queries/%s",
-                    this.baseUrl,
-                    this.apiVersion,
-                    projectId,
-                    analysisName
+            return new URL(String.format(Locale.US,
+                                         "%s/%s/projects/%s/queries/%s",
+                                         this.baseUrl,
+                                         this.apiVersion,
+                                         projectId,
+                                         analysisPath
             ));
         } catch (MalformedURLException ex) {
-            Logger.getLogger(RequestUrlBuilder.class.getName()).log(Level.SEVERE, "Failed to format query URL.", ex);
+            Logger.getLogger(RequestUrlBuilder.class.getName()).log(Level.SEVERE,
```

Comment:
Yeah I still haven't gotten my Android Studio formatting settings totally tuned yet, so sometimes I auto-format and it does weird stuff lik this. I made a note in my TODO list a few weeks ago to assess moving to an `.editorconfig` setup that works cross-IDE and will enforce simple formatting rules similar to the `.clang-format` rules you added. Should probably make that an Issue.